python/hsfs/feature_view.py

```diff
@@ -193,6 +193,7 @@ def init_serving(
         training_dataset_version: Optional[int] = None,
         external: Optional[bool] = None,
         options: Optional[dict] = None,
+        parallel=False,
```

Comment:
Why is it default parallel "False"?

Reply:
Should we not just have the number of MySQL Connections (threads) instead of a boolean?
I would prefer it to be discovered automatically - and you can override it if needed, rather than set at '1' by default.
For example, Polars doesn't give you one thread, it discovers the number of cores on your machine and sets it to that by default.

The level of parallelism we need is constrained by:
1. how many parallel lookups there will be
2. how many available MySQL connections we can use

Reply:
fixed default pool size to now use the number of prepared statements as minsize. The user can still override it by setting the minsize argument in `options` dict in `init_serving`.